packages/sdk/src/api.ts

```diff
@@ -148,6 +148,7 @@ export async function getContractSolution(
     contractCall,
     threshold,
     whitelistedSourceChains,
+    enableSwaps,
```

Comment:
@itsbobbyzzz168 @MakMuftic @mpetrunic 

To confirm, swaps work only for `/solution/call`, and for `/solutions/aggregation` should be omitted?

Reply:
yup

Reply:
got it, I removed it from this call.